src/ExtInterface/ReefMod/RMEDomain.jl

```diff
@@ -57,50 +105,56 @@ RMEDomain
 
-    site_data_path = joinpath(data_files, "region", "reefmod_gbr.gpkg")
-    site_data = GDF.read(site_data_path)
-    site_id_col = "LOC_NAME_S"
-    cluster_id_col = "LOC_NAME_S"
-    site_ids::Vector{String} = site_data[:, site_id_col]
+    # Load the geopackage file
+    gpkg_path = joinpath(data_files, "region", "reefmod_gbr.gpkg")
+    spatial_data = GDF.read(gpkg_path)
+
+    # Standardize IDs to use for site/reef and cluster
+    _standardize_cluster_ids!(spatial_data)
+
+    reef_id_col = "UNIQUE_ID"
+    cluster_id_col = "UNIQUE_ID"
 
+    reef_ids::Vector{String} = spatial_data[:, reef_id_col]
+
+    # Load accompanying ID list
+    # TODO: Create canonical geopackage file that aligns all IDs.
+    #       Doing this removes the need for the manual correction below and removes the
+    #       dependency on this file.
     id_list = CSV.read(
         joinpath(data_files, "id", "id_list_2023_03_30.csv"),
         DataFrame;
         header=false,
-        comment="#",
+        comment="#"
     )
 
-    # Re-order spatial data to match RME dataset
-    # MANUAL CORRECTION
-    site_data[site_data.LABEL_ID.=="20198", :LABEL_ID] .= "20-198"
-    id_order::Vector{Int64} = [first(findall(x .== site_data.LABEL_ID)) for x in string.(id_list[:, 1])]
-    site_data = site_data[id_order, :]
+    _manual_id_corrections!(spatial_data, id_list)
 
     # Check that the two lists of location ids are identical
-    @assert isempty(findall(site_data.LABEL_ID .!= id_list[:, 1]))
+    @assert isempty(findall(spatial_data.LABEL_ID .!= id_list[:, 1]))
 
     # Convert area in km² to m²
-    site_data[:, :area] .= id_list[:, 2] .* 1e6
+    spatial_data[:, :area] .= id_list[:, 2] .* 1e6
 
     # Calculate `k` area (1.0 - "ungrazable" area)
-    site_data[:, :k] .= 1.0 .- id_list[:, 3]
+    spatial_data[:, :k] .= 1.0 .- id_list[:, 3]
 
     # Need to load initial coral cover after we know `k` area.
-    init_coral_cover::YAXArray{Float64} = load_initial_cover(ReefModDomain, data_files, loc_ids, site_data)
+    init_coral_cover::YAXArray{Float64} = load_initial_cover(RMEDomain, data_files, loc_ids, spatial_data)
 
-    conn_data::YAXArray{Float64} = load_connectivity(ReefModDomain, data_files, loc_ids)
+    conn_data::YAXArray{Float64} = load_connectivity(RMEDomain, data_files, loc_ids)
     in_conn, out_conn, strong_pred = ADRIA.connectivity_strength(
-        conn_data, vec(site_data.area .* site_data.k), similar(conn_data)
+        conn_data, vec(spatial_data.area .* spatial_data.k), similar(conn_data)
     )
 
-    # Set all site depths to 6m below sea level
+    # Set all site depths to 7m below sea level
     # (ReefMod does not account for depth)
     # Ensure column is of float type
-    site_data[:, :depth_med] .= 6.0
-    site_data[!, :depth_med] = convert.(Float64, site_data[!, :depth_med])
+    spatial_data[:, :depth_med] .= 7.0
+    spatial_data[!, :depth_med] = convert.(Float64, spatial_data[!, :depth_med])
```

Comment:
Just noting this change from 6m to 7m is to properly align with ReefMod assumptions.